README.md

```diff
@@ -1,2 +1,42 @@
-# setup-atlas
-A GitHub Action to install ariga/atlas in your workflows.
+# setup-action
+
+A GitHub Action for installing up [Atlas](https://github.com/ariga/atlas).
+
+This action is used for installing Atlas on the Github runner.
+
+> Please note that for most use cases, you should first consider using [Atlas Action](https://github.com/ariga/atlas-action) which can save you a lot of work.
```

Comment:
`atlas-action` is used solely for CI so it is not necessarily "most cases". Instead I would say that we have other actions link [here](https://atlasgo.io/integrations/github-actions) and maybe list them.